pkg/apis/operator/v1alpha1/tektonconfig_validation.go

```diff
@@ -115,6 +115,12 @@ func (tc *TektonConfig) Validate(ctx context.Context) (errs *apis.FieldError) {
 
 	errs = errs.Also(tc.Spec.Pipeline.PipelineProperties.validate("spec.pipeline"))
 
+	errs = errs.Also(tc.Spec.Pipeline.Options.validate("spec.pipeline.options"))
+	errs = errs.Also(tc.Spec.Hub.Options.validate("spec.hub.options"))
+	errs = errs.Also(tc.Spec.Dashboard.Options.validate("spec.dashboard.options"))
+	errs = errs.Also(tc.Spec.Chain.Options.validate("spec.chain.options"))
+	errs = errs.Also(tc.Spec.Trigger.Options.validate("spec.trigger.options"))
```

Comment:
dont we need to have one for config like `errs = errs.Also(tc.Spec.Options.validate("spec.options"))`

Reply:
@piyush-garg TektonConfigSpec struct doestnt have Options

Reply:
ohk, thanks